app/models/user.js

```diff
@@ -45,6 +50,18 @@ module.exports = (sequelize, DataTypes) => {
     }
   );
 
+  User.obtainPosition = score => {
+    let position; // eslint-disable-line
+    if (score <= 5) position = 'Developer';
+    else if (score <= 9) position = 'Lead';
+    else if (score <= 19) position = 'TL';
+    else if (score <= 29) position = 'EM';
+    else if (score <= 49) position = 'HEAD';
+    else position = 'CEO';
+
+    return position;
+  };
```

Comment:
Unless you are doing some operation with the User entity, it doesn't make much sense to put this here rather than in a helper